spec.html

```diff
@@ -651,7 +581,6 @@ <h1>Expression Rules</h1>
         <emu-grammar>
           OptionalExpression :
             MemberExpression OptionalChain
-            CallExpression OptionalChain
```

Comment:
This defines tail call semantics for `return a()?.b`. You're looking for the `OptionalChainingPunctuator Arguments` below.